loops.js

```diff
@@ -10,6 +10,12 @@
 // Tip: je kunt de .repeat() methode gebruiken om een karakter een n aantal keer te herhalen...
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
+const star = '*';
+
+for (let i = 0; i < 5; i++) {
+    console.log("*".repeat(i+1));
```

Comment:
Hoi Tony, ik heb 2 opmerkingen bij deze uitwerking:
1. Je hebt wel '*' in een variabele "star"  gestopt maar 'm niet gebruikt  "star.repeat( i )"
2. Als je de conditie " i < 6 " gebruikt in de for loop is de + 1 in de repeat niet nodig